From 4a6a586d6f5ae6260879d94a6d9b420b67f066bc Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 30 Aug 2017 11:53:22 +0800
Subject: [PATCH 2466/5242] MLK-16301-1 drm/imx: dpu: plane: Take down dpu
 plane from the dpu plane grp list

commit  b82cd528456685ef953085303e2b1ceac87e6dd0 from
https://source.codeaurora.org/external/imx/linux-imx.git

No one is using the list in the dpu plane group to access dpu plane, so let's
take down dpu plane from the list so that we may remove the list entirely
from where it is defined.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dpu/dpu-plane.c |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/imx/dpu/dpu-plane.c b/drivers/gpu/drm/imx/dpu/dpu-plane.c
index 76a2cbe..9ce2930 100644
--- a/drivers/gpu/drm/imx/dpu/dpu-plane.c
+++ b/drivers/gpu/drm/imx/dpu/dpu-plane.c
@@ -484,10 +484,6 @@ struct dpu_plane *dpu_plane_init(struct drm_device *drm,
 	dpu_plane->stream_id = stream_id;
 	dpu_plane->grp = grp;
 
-	mutex_lock(&grp->lock);
-	list_add_tail(&dpu_plane->head, &grp->list);
-	mutex_unlock(&grp->lock);
-
 	plane = &dpu_plane->base;
 
 	ret = drm_universal_plane_init(drm, plane, possible_crtcs,
-- 
1.7.9.5

